seed_data/lib.py

```diff
@@ -36,54 +36,62 @@ class ModelFinder(object):
     model_cls = None
     param_keys = {}
 
-    def result(self, objects, params):
+    @classmethod
+    def result(cls, objects, params):
         """
         Processes the results from a query; returns a single model object or throws a
         relevant error if more or less than one record was found
         """
         if not params:
-            raise Exception('No parameters given for {} records. Accepted parameters: [{}]'.format(
-                self.model_cls.__name__,
-                self.param_keys
-            ))
+            raise Exception(
+                'No parameters given for {} records. Accepted parameters: [{}]'.format(
+                    cls.model_cls.__name__,
+                    cls.param_keys
+                )
+            )
         elif len(objects) == 0:
             raise Exception(
                 'No {} found with the given params ({})'.format(
-                    self.model_cls.__name__,
+                    cls.model_cls.__name__,
                     params
                 )
             )
         elif len(objects) > 1:
+            exc_text = "Multiple {} records found with the given params ({}). These parameters need to " + \
```

Comment:
Can you change the string concat to use the literal concat syntax instead, ie: `"abc" "def"` becomes `"abcdef"`